### PR TITLE
OADP-1952: Make vsc timeout configurable

### PR DESCRIPTION
**Pull request by shubham-pampattiwar**

### CODE REVIEW

File: internal/util/util.go [hunk 1]
```diff
@@ -457,12 +457,21 @@ func VSRExistsForVSB(vsb *datamoverv1alpha1.VolumeSnapshotBackup, log logrus.Fie
...
 	interval := 5 * time.Second
-	err := wait.PollImmediate(interval, timeout, func() (bool, error) {
+
+	if len(os.Getenv(DatamoverTimeout)) > 0 {
+		timeoutValue = os.Getenv(DatamoverTimeout)
+	}
+
+	timeout, err := time.ParseDuration(timeoutValue)
+	if err != nil {
+		return false, errors.Wrapf(err, "error parsing the datamover timout")
```
Comment (mateusoliveira43) [inline internal/util/util.go:471]: nit TYPO timeout

Comment (shubham-pampattiwar) [reply internal/util/util.go]: Done, fixed them

**Review by sseago — APPROVED** [review]

**Review by kaovilai — APPROVED** [review]

**Pull request merged**